### DISCUSSION TITLE
Hello from India

### DISCUSSION THREAD

**Discussion by AashiDutt** [OpenMined/courses#166] [Introduce Yourself]
Hi all, I'm a graduate in Electronics -  looking to learn new technologies and helping my career grow.
Glad to be a part of this community.

**Comment by yemikifouly** [comment]
Hello Aashi! 👋 Electronics... Can I ask what sparked your interest in AI and Private AI? And welcome to the discussion board! I hope you enjoy the course very much! 🙂

**Reply by AashiDutt** [reply]
Hey! Thanks for a warm welcome :)
So, I started working in the field of AI in the 3rd year of my college and really started to enjoy mixing up AI with Hardware. 
But recently I started my new startup with my colleague from MIT Bootcamps where we deal with lots of medical data. So, I thought this course would be really helpful for us.

**Reply by yemikifouly** [reply]
Oh wow! That is an interesting journey! 🙂 I am certain that you will enjoy this first course a lot and the following as well. This one is not technical but it is an excellent introduction to the topic. I think that you will find this course combined with the upcoming more technical ones very helpful. 🙂

**Reply by Awesome12-arch** [reply]
Hello there, I'm also from the India

**Reply by EO4wellness** [reply]
Hello to all.
